pkg/helper/dashboard.go

```diff
@@ -0,0 +1,47 @@
+package helper
+
+import "encoding/json"
+
+func FilterDashboardRows(dashboard string, ignoreFlags []string) (string, error) {
```

Comment:
So, if I'm correct, tags are duplicated: in every dashboard row (for dashboards filtering) and in metrics definitions (for FLP confgen). I guess it's ok for now but I think it could be error prone, especially as the dashboard jsons are very noisy and hard to review. So maybe we can think about how would could "secure" this, either for right now or for a follow-up. I can imagine:
- modifying FLP confgen so that it gives us the list of excluded and included metrics; then based on that, we would do the dashboards filtering based on a grep (Contains) on the promql expression
- or do the same thing without confgen, but it means the operator parsing the metrics definition files
- or, since those dashboards are statically defined, we could also just write a unit test that validates that the dashboard tags correctly match the metric defs tags

wdyt?

Reply:
btw speaking of unit test I think it would be good to have one here that ensures the dashboard files are correctly parsed without errors